lib/util/i18n.js

```diff
@@ -76,9 +105,12 @@ export async function loadLocaleData(matchedLocale, customMessages) {
       break
   }
 
+  const otpUiMessages = await loadOtpUiLocaleData(matchedLocale)
+
   // Merge custom strings into the standard language strings.
   const mergedMessages = {
     ...flatten(messages),
+    ...flatten(otpUiMessages),
```

Comment:
There might be a case where we would need OTP-RR messages to overwrite default OTP-UI messages, so ideally `otpUiMessages` should be assigned before `messages`.

Reply:
That feels good! Refactored in bfe701697e8cbceef1b004a198352f7504f4e6b0